lib/resources/example_json/r4_examples_care_team.rb

```diff
@@ -0,0 +1,128 @@
+# frozen_string_literal: true
+
+module Cerner
+  module Resources
+
+    R4_CARE_TEAM_ENTRY ||= {
+      'resourceType': 'CareTeam',
+      'id': 'SA-1316020-2076906-7016273-0-0-0',
+      'meta': {
+        'versionId': '1',
+        'lastUpdated': '2016-03-10T13:43:45.000-06:00'
+      },
+      'text': {
+        'status': 'additional',
+        'div': '&lt;div xmlns="http://www.w3.org/1999/xhtml"&gt;&lt;p&gt;&lt;b&gt;Care Team&lt;/b&gt;&lt;/p&gt;&lt;p&gt;&lt;b&gt;Patient&lt;/b&gt;: new, testfa2&lt;/p&gt;&lt;p&gt;&lt;b&gt;Status&lt;/b&gt;: Active&lt;/p&gt;&lt;p&gt;&lt;b&gt;Category&lt;/b&gt;: Encounter-focused care team&lt&lt;/p&gt;&lt;p&gt;Encounter&lt;/b&gt;: 2076906&&lt;/p&gt;lt;p&gt;&lt;b&gt;Participant&lt;/b&gt;:&lt;/p&gt;&lt;ul&gt;&lt;li&gt;Name:, Parimi Vaishnavi; Role:Respiratory Therapy&lt;/li&gt;&lt;/ul&gt;&lt;/div&gt;' # rubocop:disable Layout/LineLength
```

Comment:
render the div with < > sign instead

Reply:
Addressed. https://github.com/cerner/fhir.cerner.com/pull/470/commits/743b57306d7803dca473eb256c61982ec9c0ff62